docfx_yaml/extension.py

```diff
@@ -1268,6 +1268,45 @@ def extract_header_from_markdown(mdfile_iterator):
     return mdfile_name
 
 
+# For a given markdown file, adds syntax highlighting to code blocks.
+def highlight_md_codeblocks(mdfile):
+    find_string = '```'
+    find_string_nl = '```\n'
+    replace_string = '```python'
```

Comment:
Consider `fence`, `fence_with_nl`, and `fence_with_python`, or something like that. It's a little unclear to me while reading what these names refer to. Could just be me.

Reply:
Thank you! Didn't know they were called fence 😅